prime-router/src/main/kotlin/cli/tests/WatersAuthTests.kt

```diff
@@ -288,4 +291,54 @@ class WatersAuthTests : CoolTest() {
         teardown()
         return passed
     }
+}
+
+/**
+ * Exercise the database jticache
+ */
+class Jti : CoolTest() {
```

Comment:
This does not seem to be related to a waters auth test.  Can you move to another file?

Reply:
Yes, it's related.   The JTI cache is used by the auth code to prevent a man-in-the middle attack.